src/Vision/VNUtils.cs

```diff
@@ -54,6 +54,22 @@ public static partial class VNUtils {
 		[DllImport (Constants.VisionLibrary, EntryPoint = "VNNormalizedRectForImageRect")]
 		public static extern CGRect GetNormalizedRect (CGRect imageRect, nuint imageWidth, nuint imageHeight);
 
+		[TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[DllImport (Constants.VisionLibrary, EntryPoint="VNImagePointForNormalizedPointUsingRegionOfInterest")]
+		public static extern CGPoint GetImagePoint (CGPoint normalizedPoint, nuint imageWidth, nuint imageHeight, CGRect roi);
+
+		[TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
```

Comment:
You need to add the .NET version of the attributes as well (there's an example a little up in this file)